src/archanaNJun20/ReplaceTechnoWithCredits.java

```diff
@@ -0,0 +1,39 @@
+package archanaNJun20;
+
+import java.util.ArrayList;
+
+public class ReplaceTechnoWithCredits{
+	ArrayList<String> replaceTechnoWithCredits(ArrayList<String> input){
+		System.out.print("Techno word index in given arrayList is  ");
+		for (int index = 0; index < input.size(); index++){
+			if (input.contains("Techno")){
+				System.out.print(input.indexOf("Techno") + ",");
+				input.set(input.indexOf("Techno"), "Credits");
+			}
+		}
+		System.out.println();
+		return input;
+	}
+	void countWord(ArrayList<String> input){
+		int count=0;
+		for (int index = 0; index <= input.size(); index++){
```

Comment:
here also you can write like this "while(input.contains("Credits"))", as well as in above method "replaceTechnoWithCredits(ArrayList<String> input)"